### ISSUE TITLE
VAmiga crashes on closing

### ISSUE THREAD

**Issue by bluecursor** [dirkwhoffmann/vAmiga#473]
Attached is the full error message

[error.txt](https://github.com/dirkwhoffmann/vAmiga/files/5976336/error.txt)

**Comment by dirkwhoffmann** [comment]
Confirmed. I experienced the very same error today:

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   dirkwhoffmann.vAmiga          	0x000000010be4ddc4 Swift runtime failure: Unexpectedly found nil while implicitly unwrapping an Optional value + 0 (MyControllerStatusBar.swift:133) [inlined]
1   dirkwhoffmann.vAmiga          	0x000000010be4ddc4 MyController.refreshStatusBar(writing:) + 1044 (MyControllerStatusBar.swift:133)
2   dirkwhoffmann.vAmiga          	0x000000010be6705b closure #1 in closure #1 in MyController.addListener() + 8 (MyController.swift:312) [inlined]
3   dirkwhoffmann.vAmiga          	0x000000010be6705b partial apply for closure #1 in closure #1 in MyController.addListener() + 27
4   dirkwhoffmann.vAmiga          	0x000000010be6096e thunk for @escaping @callee_guaranteed () -> () + 14
5   libdispatch.dylib             	0x00007fff201685dd _dispatch_call_block_and_release + 12
6   libdispatch.dylib             	0x00007fff201697c7 _dispatch_client_callout + 8
7   libdispatch.dylib             	0x00007fff20175b86 _dispatch_main_queue_callback_4CF + 940
8   com.apple.CoreFoundation      	0x00007fff20449970 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
9   com.apple.CoreFoundation      	0x00007fff2040b852 __CFRunLoopRun + 2731
10  com.apple.CoreFoundation      	0x00007fff2040a6ce CFRunLoopRunSpecific + 563
11  com.apple.HIToolbox           	0x00007fff28692630 RunCurrentEventLoopInMode + 292
12  com.apple.HIToolbox           	0x00007fff2869242c ReceiveNextEventCommon + 709
13  com.apple.HIToolbox           	0x00007fff2869214f _BlockUntilNextEventMatchingListInModeWithFilter + 64
14  com.apple.AppKit              	0x00007fff22c2a9b1 _DPSNextEvent + 883
15  com.apple.AppKit              	0x00007fff22c29177 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1366
16  com.apple.AppKit              	0x00007fff22c1b68a -[NSApplication run] + 586
17  com.apple.AppKit              	0x00007fff22bef96f NSApplicationMain + 816
18  dirkwhoffmann.vAmiga          	0x000000010be17b89 main + 9 (MyAppDelegate.swift:18)
19  libdyld.dylib                 	0x00007fff2032f621 start + 1

```

**Comment by dirkwhoffmann** [comment]
Hopefully fixed in v0.9.18 which utilises a different shutdown sequence.
